src/programs/ectrans-benchmark.F90

```diff
@@ -361,7 +358,6 @@ program transform_test
     numll(jroc) = ilevpp
   endif
 enddo
-numll(iprused+1:nprtrv+1) = 0
```

Comment:
All indices passed to `numll` are in the bounds `1:nprtrv`. The only one that isn't obvious is:
```
nflevl = numll(mysetv)
```
but `mysetv` is the index in the V set, so must be <= `nprtrv`. So I think it's safe to trim `numll`.

Reply:
So trimming numll to (1:nprtrv) is OK and I understand.
But removal of  `numll(iprused+1:nprtrv) = 0`, is that OK then?
That is OK in case `iprused >= nprtrv`, and is that correct?

Reply:
`iprused` is either equal to `nprtrv` or `nflevg+1`.

If it's `nprtrv`, then this line simply sets the last element, the `nprtrv+1`th, to zero. But that element is never accessed.

If it's `nflevg+1`, it's more complicated. Then, `ilevpp` is `0` and `irest` is `nflevg`. As a consequence, in the following loop all elements with an index `> nflevg` are set to `0` (because `ilevpp = 0`). This line simply does this again, so is redundant. The only element it sets to `0` which isn't already is the `nprtrv+1`th. Again...